src/utils/epochWeekNumber.ts

```diff
@@ -108,9 +108,9 @@ export const epochWMToCalendarWeekNumber = (epochWeekNumber: number): number =>
   // Set Day to Sunday and make it the 7th day of the week
   date.setUTCDate(date.getUTCDate() + 4 - (date.getUTCDay()||7));
   // Get first day of year
-  var yearStart = new Date(Date.UTC(date.getUTCFullYear(),0,1));
+  let yearStart = new Date(Date.UTC(date.getUTCFullYear(),0,1));
```

Comment:
constante tant qu'à faire

```suggestion
  const yearStart = new Date(Date.UTC(date.getUTCFullYear(), 0, 1));
```